azure-cron.yml

```diff
@@ -101,10 +103,34 @@ jobs:
           aws cloudfront create-invalidation \
                          --distribution-id E1U753I56ERH55 \
                          --paths '/*'
+          echo "Publishing $LATEST to Hubspot..."
+          DESCRIPTION_MD="$(cat $GH_RESP | jq -r '. | map(select(.tag_name="'$LATEST'"))[0].body')"
+          DESCRIPTION=$(curl -s -XPOST 'https://api.github.com/markdown/raw' -H 'Content-Type: text/plain' -d"$DESCRIPTION_MD" | jq -sR .)
+          DATE="$(date -d $(cat $GH_RESP | jq -r '. | map(select(.tag_name="'$LATEST'"))[0].published_at') +%s)000"
+          BLOG=11411412838
```

Comment:
Where is this magic number coming from? Is it the ID of our blog? Same for `blog_author_id` below/

Reply:
Yes, id of our blog and blog author, respectively. I've added a line of comment to clarify.